tests/playwright.config.ts

```diff
@@ -30,4 +30,7 @@ export default defineConfig({
       use: { ...devices['Desktop Safari'] },
     },
   ],
+  expect: {
+    timeout: 15 * 1000,
```

Comment:
nit: in the Playwright docs, they used the value of `15_000` instead of `15 * 1000`. Might be considering to change this one..

Ref: https://playwright.dev/docs/test-timeouts

and also, why in the commit it's said `increase individual action timeout from 5s to 10s` but here it's specified as `15s`?

Reply:
Fixed